### PR TITLE
RSDEV-408 Fix failing tests

### PR DESCRIPTION
**Pull request by rlamacraft**
Would be good if someone else could run these tests too as I don't believe we have any automated CI -- just in case there's anything odd about my machine

### CODE REVIEW

**Review by nebay-abraha — APPROVED** [review]
I have run tests against rspace `v2.4.0` and can confirm fix working as expected.

**Comment by rlamacraft** [comment]
> I have run tests against rspace `v2.4.0` and can confirm fix working as expected.

Brilliant, thanks

**Review by mKowalski256 — APPROVED** [review]
Looks good - when running `invapi_test.py` script from this branch I'm seeing `1 failed, 55 passed`, compared to `7 failed, 49 passed` in `master` branch.

File: rspace_client/tests/invapi_test.py [hunk 1]
```diff
@@ -286,7 +286,7 @@ def test_duplicate(self):
 
     def test_get_benches(self):
         benches = self.invapi.get_workbenches()
-        self.assertEqual(1, len(benches))
+        self.assertEqual(2, len(benches))
```
[the remark this answers is not in the document]
Comment (rlamacraft) [reply rspace_client/tests/invapi_test.py:289]: Yeah I was using user1a who can see user2b's bench. All of these tests are pretty fragile as they depend on the database contents of a particular instance and don't do as much setup as the main product's e2e tests.

**Pull request merged**